python/ray/dag/compiled_dag_node.py

```diff
@@ -75,10 +73,21 @@ def do_exec_compiled_task(
                 channel.end_read()
 
     except Exception as e:
-        logging.warn(f"Compiled DAG task aborted with exception: {e}")
+        logging.info(f"Compiled DAG task exited with exception: {e}")
```

Comment:
For non-Ray exceptions, I wonder if we should instead store the error and keep looping?

Reply:
+1